config/default/configmap/kfservice.yaml

```diff
@@ -34,5 +34,8 @@ data:
        "s3": {
            "s3AccessKeyIDName": "awsAccessKeyID",
            "s3SecretAccessKeyName": "awsSecretAccessKey"
+       },
+       "azure": {
+           "azureSecretName": "azcreds"
```

Comment:
I'm not sure if here should be secret name, personally I think should be name of secret.data.key? Please @yuzisun confirm.

Reply:
The secret has a bunch of keys... I dont want to pick the first secret that happens to have an expected key...

Reply:
discussed offline, its not a good practice to share SP secrets... we will reevaluate if users are confused and raise issues